src/syntax/ppx_bisect.ml

```diff
@@ -0,0 +1,13 @@
+open Migrate_parsetree
+open Ppx_tools_404
+
+let () =
+  Driver.register ~name:"ppx_bisect" ~args:InstrumentArgs.switches
+    Versions.ocaml_404 (fun _config _cookies ->
+        (* let anon s = raise (invalid_arg ("nothing anonymous: " ^ s)) in *)
+        let anon _s = () in
+        let usage = Printf.sprintf "Usage: bisect_ppx <options>" in
+        let arga = Array.of_list ("" :: (Array.to_list Sys.argv)) in
+        Arg.parse_argv arga InstrumentArgs.switches anon usage;
+        Ast_mapper_class.to_mapper (new InstrumentPpx.instrumenter)
```

Comment:
Yes, the purpose of exposing Arg.specs to the driver is that the rewriter doesn't deal with arg parsing by itself (so that multiple rewriters are more likely to compose).

So you shouldn't do `Arg.parse_argv`, when the rewriting function is called Arg.specs have already been executed.